--- ../plantuml/src/main/java/net/sourceforge/plantuml/klimt/creole/SheetBlock1.java
+++ ../plantuml/src/main/java/net/sourceforge/plantuml/klimt/creole/SheetBlock1.java
@@ -107,6 +107,7 @@
 		for (Stripe stripe : stripes) {
 			if (stripe.getAtoms().size() == 0)
 				continue;
 
 			final Sea sea = new Sea(stringBounder);
 			for (Atom atom : stripe.getAtoms())
@@ -118,6 +119,7 @@
 			final double width = sea.getWidth();
 			widths.put(stripe, width);
 			minMax = sea.update(minMax);
 			final double height = sea.getHeight();
+			System.err.println("SHEETBLOCK STRIPE: width=" + width + " height=" + height + " atoms=" + stripe.getAtoms().size());
 			heights.put(stripe, height);
 			y += height;
 			if (stringBounder.matchesProperty("TIKZ")) {
